common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -28,10 +45,12 @@ object ImageMetadataConverter {
       subLocation         = fileMetadata.iptc.get("Sub-location"),
       city                = fileMetadata.iptc.get("City"),
       state               = fileMetadata.iptc.get("Province/State"),
-      country             = fileMetadata.iptc.get("Country/Primary Location Name")
+      country             = fileMetadata.iptc.get("Country/Primary Location Name"),
+      suppliersCategory   = fileMetadata.iptc.get("Category")
```

Comment:
My hunch having looked at some data is that both Category and Supplemental Category are a bit of a mess, using two standards (single and three letter codes) and sometimes full words. It's not clear that the Category is reliably a different or more important category than the Supplemental Category.

My instinct would be to bundle them all into a single list -- simpler and I doubt users will ever need the power of major vs minor categories.

If it's too much faff to map many categories, it probably makes sense to start with the sport (or sport related) ones as you've done, and add more later. It would however be easier to get the ES mapping right the first time around, as an ES reindex is still not massively fun.